front/src/modules/command-menu/components/CommandItemText.tsx

```diff
@@ -0,0 +1,45 @@
+import styled from '@emotion/styled';
+
+export type CommandItemTextProps = {
+  text: string;
+};
+
+const StyledCommandTextContainer = styled.div`
+  display: flex;
+  gap: ${({ theme }) => theme.spacing(1)};
+  justify-content: center;
+`;
+
+const StyledCommandText = styled.div`
+  color: ${({ theme }) => theme.font.color.light};
+  padding-left: ${({ theme }) => theme.spacing(2)};
+  padding-right: ${({ theme }) => theme.spacing(2)};
+  white-space: nowrap;
+`;
+
+const StyledCommandKeys = styled.div`
```

Comment:
This could match the design better : https://www.figma.com/file/xt8O9mFeLl46C5InWwoMrN/Twenty?type=design&node-id=14549-47927&mode=design&t=89WsRl8iB25MjXpq-4